tests/integration/validations/factory-dependent-keys-test.js

```diff
@@ -63,7 +63,8 @@ module('Integration | Validations | Factory - Dependent Keys', function(hooks) {
     );
   });
 
-  test('nested ds-error validator creates correct dependent keys', function(assert) {
+  // ember-validators v4.0.0 broke the ds-error validator for nested objects
```

Comment:
This isn't an issue?

Reply:
I've run this change through our main project and it looks fine. For other projects using this fork, we're on a major release here (as well as beta), that suggests breaking changes.

Reply:
Ok, I trust you. :+1: